setup.py

```diff
@@ -90,10 +90,12 @@
         'requests>=2.25.1,<3.0',
         'rfc3986-validator>=0.1.1',
         'tornado>=6.1.0',
+        'typing-extensions>=3.10,<4',  # Cap from KFP
         'traitlets>=4.3.2',
         'urllib3>=1.26.5',
         'watchdog>=2.1.3',
         'websocket-client',
+        'wrapt>=1.11,<1.14',  # Add from asteroid
         'yaspin',
         # KFP runtime dependencies
         'kfp>=1.7.0,<2.0,!=1.7.2',
```

Comment:
Can we also add a comment here?  The model is that anything capped must be on our radar to uncap as soon as possible.

Reply:
comment added